src/ess/powder/conversion.py

```diff
@@ -175,7 +178,7 @@ def _restore_tof_if_in_wavelength(data: sc.DataArray) -> sc.DataArray:
 
 
 def add_scattering_coordinates_from_positions(
-    data: NormalizedByProtonCharge[RunType], graph: ElasticCoordTransformGraph
```

Comment:
I'm not sure I managed to follow why this is now `FilteredData`... I would have expected `NormalizedRunData`?

Reply:
Monitor normalization needs a wavelength coord. So it has to be done after `add_scattering_coordinates_from_positions`. For proton charge normalization, the order does not matter. I moved it to the same place as monitor normalization to make it easier to build the graph.